tests/functional/elements.js

```diff
@@ -2,27 +2,234 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-const elements = {
-    MULTIHOP_SELECTOR_TAB: "multiHopSelector/tabMultiHop",
-    SINGLEHOP_SELECTOR_TAB: "multiHopSelector/tabSingleHop",
-    SERVERLIST_BACK_BUTTON: "serverListBackButton",
-    SERVER_ENTRY_BUTTON: "buttonSelectEntry",
-    SERVER_EXIT_BUTTON: "buttonSelectExit",
-    SERVER_LIST_BUTTON: "serverListButton",
-    SERVER_COUNTRY_VIEW: "serverCountryView",
+ const generalElements = {     
     CONTROLLER_TITLE: "controllerTitle",
-    CITYLIST_VISIBLE: "cityListVisible",
-    VPN_MULTHOP_CHEVRON: 'vpnCollapsibleCardChevron',
-    VPN_COLLAPSIBLE_CARD: 'vpnCollapsibleCard',
+    CONTROLLER_SUBTITLE: 'controllerSubTitle',
+    CONTROLLER_TOGGLE: 'controllerToggle'
+}
+
+const elementState = {
+    COUNTRY_VIEW: "serverCountryView",
+    CITYLIST_VISIBLE: "cityListVisible",   
+}
+
+const navBar = {
+    NAVIGATION_BAR: 'navigationBar',
+    HOME: 'navigationLayout/navButton-home',
+    MESSAGES: 'navigationLayout/navButton-messages',
+    SETTINGS: 'navigationLayout/navButton-settings',
+}
+
+const telemetryView = {    
+    TELEMETRY_POLICY_BUTTON: 'telemetryPolicyButton',
+    DECLINE_TELEMETRY: 'declineTelemetryLink',
+    POST_AUTHENTICATION_BUTTON: 'postAuthenticationButton'    
+}
+
+/*
+    Settings screen and related screens
+*/
+const settingsView = {
+    SETTINGS: 'settingsView',    
+    SCREEN: 'settingsView-flickable', // or SETTINGS_VIEW: 'settingsView',
+    USER_PROFILE: 'settingsUserProfile-manageAccountButton',
+    TIPS_AND_TRICKS: 'settingsTipsAndTricks',
+    NETWORK_SETTINGS: 'settingsNetworking',
+    SYSTEM_PREFERENCE: 'settingsPreferences',
+    MY_DEVICE: 'settingsDeviceList',
+    GET_HELP: 'settingsGetHelp',
+    ABOUT_US: 'settingsAboutUs',
+    SIGN_OUT: 'settingsLogout',
+    BACK: 'settings-back',
+
+
+    // GET HELP VIEW
+    getHelpView: {
```

Comment:
well... getHelp is a screen. should go to the top-level. For instance, getHelp can be opened from the initial screen before the authentication too.